inst/templates/common/header.mustache

```diff
@@ -10,7 +10,9 @@
 <div class="header-labels">
 <div class="header-col-left">
 <p><strong>Location:</strong> {{reportMetadata.stationId}} - {{reportMetadata.stationName}}</p>
+<!-- Only DVHydro and 5YR GW use primarySeriesLabel -->
 {{#reportMetadata.primarySeriesLabel}}<p><strong>Primary Time Series:</strong> {{reportMetadata.primarySeriesLabel}}</p>{{/reportMetadata.primarySeriesLabel}}
+{{#reportMetadata.timeseriesLabel}}<p><strong>Primary Time Series:</strong> {{reportMetadata.timeseriesLabel}}</p>{{/reportMetadata.timeseriesLabel}}
```

Comment:
It's all over the place: DvHydro uses primarySeriesLabel, and SRS, SVP and DC use timeseriesLabel, Extremes uses primaryLabel, corrections and UvHydro uses primaryParameter.  I can change derivation chain, but if we go to whisker templates for all reports, a lot of the other reports will need to be changed.